app/models/accessible_format_request.rb

```diff
@@ -0,0 +1,19 @@
+class AccessibleFormatRequest
+  include ActiveModel::Validations
+  attr_accessor :content_id, :attachment_id, :format_type, :custom_details, :contact_name, :contact_email
+
+  validates :content_id, presence: true
+  validates :attachment_id, presence: true
+  validates :format_type, presence: true
+  validates :contact_name, presence: true
+  validates :contact_email,
+            presence: true,
+            email: { message: "The email address must be valid" },
+            length: { maximum: 1250 }
+
+  def initialize(attributes = {})
+    attributes.each do |key, value|
+      send("#{key}=", value) if respond_to? "#{key}="
```

Comment:
this will only attempt to assign an attribute if the attribute actually exists on the model.
It's unlikely to be needed as we also control the model creation in the controller, so happy to remove it if it's more confusing